packages/dashboard-core-plugins/src/panels/IrisGridPanel.tsx

```diff
@@ -1212,10 +1226,7 @@ export class IrisGridPanel extends PureComponent<
       this.getPluginContent(Plugin, model, user, workspace, pluginState);
     const { permissions } = user;
     const { canCopy, canDownloadCsv } = permissions;
-    const formattedRowCount = model?.displayString(
-      model?.rowCount ?? 0,
-      'long'
-    );
+    const formattedRowCount = model?.displayString(rowCount, 'long');
```

Comment:
rowCount could be undefined here, use `?? 0` or set a default value when pulling it from state at top of render function.